docs/topics/logging.rst

```diff
@@ -193,6 +193,15 @@ to override some of the Scrapy settings regarding logging.
     Module `logging.handlers <https://docs.python.org/2/library/logging.handlers.html>`_
         Further documentation on available handlers
 
+Custom Log Formats
+-------------------
+
+Custom log format can be set for different actions by extending :class:`~scrapy.logformatter.LogFormatter` class
```

Comment:
Don’t forget to use [articles](https://dictionaryblog.cambridge.org/2015/08/19/a-an-and-the-how-to-use-articles-in-english/).

```suggestion
A custom log format can be set for different actions by extending the :class:`~scrapy.logformatter.LogFormatter` class
```